sound/soc/intel/boards/sof_sdw.c

```diff
@@ -892,6 +900,7 @@ static struct sof_sdw_codec_info codec_info_list[] = {
 				.dai_type = SOF_SDW_DAI_TYPE_AMP,
 				.dailink = {SDW_AMP_OUT_DAI_ID, SDW_UNUSED_DAI_ID},
 				.init = sof_sdw_maxim_init,
+				.rtd_init = maxim_spk_rtd_init,
```

Comment:
... here we have both callbacks. I really can't follow which callback does what.